endpoints-support.md

```diff
@@ -21,8 +21,8 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
 <br />
```

Comment:
Can you please check the formatting of this file.

Reply:
checking it already

Reply:
Done.